pkg/csi/service/vanilla/controller.go

```diff
@@ -105,22 +105,22 @@ func (c *controller) Init(config *cnsconfig.Config, version string) error {
 	ctx, log := logger.GetNewContextWithLogger()
 	log.Infof("Initializing CNS controller")
 	var err error
-	var operationStore cnsvolumeoperationrequest.VolumeOperationRequest
-	operationStore, err = cnsvolumeoperationrequest.InitVolumeOperationRequestInterface(ctx,
-		config.Global.CnsVolumeOperationRequestCleanupIntervalInMin,
-		func() bool {
-			return commonco.ContainerOrchestratorUtility.IsFSSEnabled(ctx, common.BlockVolumeSnapshot)
-		})
-	if err != nil {
-		log.Errorf("failed to initialize VolumeOperationRequestInterface with error: %v", err)
-		return err
-	}
 	multivCenterCSITopologyEnabled = commonco.ContainerOrchestratorUtility.IsFSSEnabled(ctx,
 		common.MultiVCenterCSITopology)
 	isAuthCheckFSSEnabled := commonco.ContainerOrchestratorUtility.IsFSSEnabled(ctx, common.CSIAuthCheck)
 
 	vcManager := cnsvsphere.GetVirtualCenterManager(ctx)
 	if !multivCenterCSITopologyEnabled {
+		var operationStore cnsvolumeoperationrequest.VolumeOperationRequest
+		operationStore, err = cnsvolumeoperationrequest.InitVolumeOperationRequestInterface(ctx,
+			config.Global.CnsVolumeOperationRequestCleanupIntervalInMin,
+			func() bool {
+				return commonco.ContainerOrchestratorUtility.IsFSSEnabled(ctx, common.BlockVolumeSnapshot)
+			})
+		if err != nil {
+			log.Errorf("failed to initialize VolumeOperationRequestInterface with error: %v", err)
+			return err
+		}
```

Comment:
IIUC, `InitVolumeOperationRequestInterface` is not doing anything specific to VC. Do we need a separate instance of `operationStore` for every VC?

Reply:
No, one instance is enough. Made required change.